pkg/packagekit/updates.scss

```diff
@@ -6,6 +6,13 @@
     @extend .ct-card;
 }
 
+@media screen and (max-width: 1200px) {
+        // Badges should not stretch in mobile/tablet mode
+       .pf-c-table [data-label] > .pf-c-badge {
+           justify-self: start;
+       }
```

Comment:
If this is moved more upstream, then it might need to handle other types of widgets too, other than just badges. For example, the badly-named ["label" component](https://www.patternfly.org/v4/components/label) is basically the same thing as a badge, but has a different name.

There are _probably_ other components that shouldn't be stretched too, but both badge/label are obvious ones that shouldn't be stretched in mobile mode.

HOWEVER, we might want to consider a so-called ["label" in a basic label group](https://www.patternfly.org/v4/components/label-group#basic) (of one), where the group is stretched but the labels aren't.

Then we probably wouldn't need any custom CSS, at the expense of the DOM becoming a little bit more needlessly complex.